resources/public/api/conf/6.0/uk_property_accounting_adjustments_submit.yaml

```diff
@@ -48,24 +48,36 @@ post:
           oneOf:
             - $ref: "./schemas/submit_uk_property_accounting_adjustments/def1/request.json"
             - $ref: "./schemas/submit_uk_property_accounting_adjustments/def2/request.json"
+            - $ref: "./schemas/submit_uk_property_accounting_adjustments/def3/request.json"
         examples:
           example-1:
```

Comment:
I know that these are called `examples-x` in the previous version but shouldn't these scenarios have names to specify what the request body is (like the retrieve endpoint)? 
![image](https://github.com/user-attachments/assets/268500b2-4139-49a8-a028-d94629e097a7)